pkg/node/node.go

```diff
@@ -223,3 +223,31 @@ func (n *Node) ClusterDNS() string {
 	}
 	return "10.100.0.10"
 }
+
+const (
+	Arm64 = "arm64"
+	Amd64 = "x86_64"
+)
+
+func (n *Node) PauseImage() string {
+	var account, arch string
+	switch n.Region {
+	case "ap-east-1":
+		account = "800184023465"
```

Comment:
What account IDs are these and where are they coming from? Maybe it would be better to fetch these from somewhere instead of hardcoding them, or at least adding a comment with the pertinent doc's URL?

Reply:
https://github.com/awslabs/amazon-eks-ami/blob/master/files/bootstrap.sh#L103-L113

This doesn't seem to be documented elsewhere, but they are also to be found in the eksctl sourcecode https://github.com/weaveworks/eksctl/blob/master/pkg/apis/eksctl.io/v1alpha5/types.go#L169-L177

Reply:
Seems to be an AWS account just to hold those AMIs: https://github.com/terraform-providers/terraform-provider-aws/commit/7c88258e45ce6ffa633a850629b4a146460c89c8

Reply:
Yes ... in this case we want to use these accounts for the ECR address where they store the pause image... so we don't have to download that from gcr or dockerhub or somesuch!